sources/packages/backend/apps/queue-consumers/src/services/application/application.service.ts

```diff
@@ -1,21 +1,24 @@
 import { Injectable } from "@nestjs/common";
-import { DataSource } from "typeorm";
+import { Brackets, Repository } from "typeorm";
 import {
-  RecordDataModelService,
   Application,
   ApplicationStatus,
   User,
+  StudentAssessment,
+  StudentAssessmentStatus,
 } from "@sims/sims-db";
 import { ConfigService } from "@sims/utilities/config";
+import { InjectRepository } from "@nestjs/typeorm";
 
 @Injectable()
-export class ApplicationService extends RecordDataModelService<Application> {
+export class ApplicationService {
   constructor(
-    dataSource: DataSource,
     private readonly configService: ConfigService,
-  ) {
-    super(dataSource.getRepository(Application));
```

Comment:
Why did you remove the "default" repo here?

Reply:
It was a minor refactor. Based on past conversations the idea was to prefer the syntax using `@InjectRepository(Application)` instead of the one being refactored (we can review the benefits if needed).
The services under queue-consumers are following it and the idea is follow on every new class.